tracer/build/_build/Build.GitHub.cs

```diff
@@ -986,6 +986,29 @@ await client.Issue.Milestone.Update(
              await ReplaceCommentInPullRequest(prNumber, $"## Benchmarks Report for " + BenchmarkCategory, markdown);
          });
 
+    Target CompareBenchmarksResultsBP => _ => _
+         .Unlisted()
+         .DependsOn(CreateRequiredDirectories)
+         .Requires(() => GitHubRepositoryName)
+         .Requires(() => BenchmarkCategory)
+         .Executes(() =>
+         {
+             if (!int.TryParse(Environment.GetEnvironmentVariable("PR_NUMBER"), out var prNumber))
+             {
+                 Logger.Warning("No PR_NUMBER variable found. Skipping benchmark comparison");
+                 return;
+             }
+
+             var masterDir = BuildDataDirectory / "previous_benchmarks";
+             var prDir = BuildDataDirectory / "benchmarks";
+
+             var markdown = CompareBenchmarks.GetMarkdown(masterDir, prDir, prNumber, "master", GitHubRepositoryName, BenchmarkCategory);
+             string tempFolder = Path.GetTempPath();
+             string filePath = Path.Combine(tempFolder, "benchmarks_report.md");
```

Comment:
I like that. 
Just a tiny suggestion:
```suggestion
             string filePath = Path.Combine(Path.GetTempPath(), "benchmarks_report.md");
```
As `tempFolder` is only used once and `Path.GetTempPath()` is quite self-explainatory